docs/release.md

```diff
@@ -15,20 +15,34 @@
   - `git push`
   - Create a PR for this change, merge once approved.
 4. Create an internal release tag to mark on the master branch where we forked for the release.
-  - `git tag "${RELEASE}.x"`
-  - `git push origin "${RELEASE}.x"`
+  - `git tag "${COLLECTOR_RELEASE}.x"`
+  - `git push origin "${COLLECTOR_RELEASE}.x"`
 5. Create the release branch with an empty commit and push.
-  - `git checkout -b "release/${RELEASE}.x"`
-  - `git commit --allow-empty -m "Empty commit to diverge ${RELEASE} from master"`
-  - `git push --set-upstream origin "release/${RELEASE}.x"`
+  - `git checkout -b "release-${COLLECTOR_RELEASE}"`
+  - `git commit --allow-empty -m "Empty commit to diverge ${COLLECTOR_RELEASE} from master"`
+  - `git push --set-upstream origin "release-${COLLECTOR_RELEASE}"`
 
 **Tag and create the collector image release**
 
 1. Increment or set the patch number and release environment variables (if not set).
-  - `export PATCH_NUMBER=0`
-  - `export RELEASE=3.8`
-2. Tag and push the release. Check CI to ensure the triggered builds complete.
-  - `git tag "${RELEASE}.${PATCH_NUMBER}"`
-  - `git push origin "${RELEASE}.${PATCH_NUMBER}"`
-3. Create a pull request to update the `COLLECTOR_VERSION` file in the [stackrox/stackrox](https://github.com/stackrox/stackrox/) repo with the newly create release after CI images have been built.
+  - `export COLLECTOR_PATCH_NUMBER=0`
+  - `export COLLECTOR_RELEASE=3.8`
+2. Tag and push the release.
+  - `git tag "${COLLECTOR_RELEASE}.${COLLECTOR_PATCH_NUMBER}"`
+  - `git push origin "${COLLECTOR_RELEASE}.${COLLECTOR_PATCH_NUMBER}"`
+3. Create a corresponding PR in openshift/release. Go to a forked copy of the openshift/release repo
```

Comment:
I think that the openshift/release steps should appear before the tagging instructions. For each release branch we will create the branch and openshift/release config *once* and then we can create multiple PRs for proposed cherry-picks if needed and multiple tags.

Reply:
If we made the openshift/release release branch first it would push an image with SHA instead of the release tag, so I don't think that is correct.